### ISSUE TITLE
Using rust.vim with Vim 7.4.2143 results in W19 being triggered

### ISSUE THREAD

**Issue by YorickPeterse** [rust-lang/rust.vim#96]
Vim 7.4.something added a new feature where it adds a warning in certain cases when defining/undefining augroups: https://groups.google.com/forum/#!topic/vim_dev/PtIxKYsCEos

This particular warning is triggered by rust.vim whenever you open a Rust file. I found out I can work around this by applying the following patch:

``` diff
diff --git a/ftplugin/rust.vim b/ftplugin/rust.vim
index 7d27c85..89138e9 100644
--- a/ftplugin/rust.vim
+++ b/ftplugin/rust.vim
@@ -188,7 +188,6 @@ let b:undo_ftplugin = "
                \|ounmap <buffer> ]]
                \|set matchpairs-=<:>
                \|unlet b:match_skip
-               \|augroup! rust.vim
                \"

 " }}}1
```

I doubt this is the right way of actually solving it, but at the same time I'm not really sure what the proper solution _would_ be.

To test this:
1. Upgrade to the latest Vim
2. Install rust.vim using your favourite plugin manager
3. Open an existing Rust File using `vim some_file.rs -c 'sleep 30'` (the sleep is added so the actual warning doesn't get suppressed right away)

You'll then see something like this:

![vim](https://cloud.githubusercontent.com/assets/86065/17572065/1ee97024-5f53-11e6-8155-aeed3a1d83de.jpg)

Whenever this warning pops up you have to explicitly press the Enter key to continue opening the file.

**Comment by YorickPeterse** [comment]
Documentation on the new warning: https://github.com/vim/vim/blob/bc8801c9317eb721a2ee91322669f2dd5d136380/runtime/doc/autocmd.txt#L1166

**Comment by euclio** [comment]
Should be fixed by #124.

**Comment by da-x** [comment]
Please reopen if the issue persists. Thanks!
